src/main/java/org/sciborgs1155/robot/vision/VisionConstants.java

```diff
@@ -0,0 +1,46 @@
+package org.sciborgs1155.robot.vision;
+
+import edu.wpi.first.apriltag.AprilTagFieldLayout;
+import edu.wpi.first.apriltag.AprilTagFields;
+import edu.wpi.first.math.Matrix;
+import edu.wpi.first.math.VecBuilder;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Transform3d;
+import edu.wpi.first.math.numbers.N1;
+import edu.wpi.first.math.numbers.N3;
+import org.sciborgs1155.robot.vision.Vision.CameraConfig;
+
+public class VisionConstants {
+  public static final AprilTagFieldLayout TAG_LAYOUT =
+      AprilTagFieldLayout.loadField(AprilTagFields.kDefaultField);
+
+  /* The front of the robot is defined at the intake */
+  public static final CameraConfig BACK_LEFT_CAMERA =
+      new CameraConfig("back left", new Transform3d());
```

Comment:
maybe make a note about needing to find these

actually, i think we should either have a bunch of Todo comments, or in our readme have a list of the things that you'll need to change depending on your robot (or both!)